src/main/java/org/broadinstitute/hellbender/tools/spark/bwa/BwaSpark.java

```diff
@@ -23,14 +23,23 @@
 
     private static final long serialVersionUID = 1L;
 
+    public static final String SINGLE_END_ALIGNMENT_FULL_NAME = "singleEndAlignment";
+    public static final String SINGLE_END_ALIGNMENT_SHORT_NAME = "SE";
+
     @Argument(doc = "the output bam", shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
             fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME)
     private String output;
 
     @Argument(doc = "the bwa mem index image file name that you've distributed to each executor",
-            fullName = "bwamemIndexImage")
+            fullName = "bwaMemIndexImage")
```

Comment:
This should be extracted as a static final `String` for visibility (and maybe re-use).